### DISCUSSION TITLE
Vypnout aplikaci

### DISCUSSION THREAD

**Discussion by jaro-jaro** [jaro-jaro/DPMCB#136] [Ideas]
Navrhuju, že by se dalo v nastavení aplikace dalo vypnout hlášení při vypínání aplikace, ale pokaždý, když by aplikaci někdo vyzkoušel zavřít, tak by se mu dal toast nebo dialog, že je ničitel, tich toustů by mohlo bejt víc po sobě, ať to chvilku trvá

**Comment by rblaha15** [comment]
Zajímavé ....

**Comment by rblaha15** [comment]
Nevím, co si o tom myslet

**Reply by 0ndra0** [reply]
šlo by vypínat aplikaci i tiše a přidali bysme ničitele

**Reply by rblaha15** [reply]
Tak jo no :)

**Comment by rblaha15** [comment]
- \>\>\> #154
